### DISCUSSION TITLE
diacritics on the spokes of the wheel

### DISCUSSION THREAD

**Discussion by jakamilek** [8VIM/8VIM#306] [Q&A] [answered]
Why are the special characters (diacritics) not displayed on the spokes of the wheel when selecting?
Cheers

Ps I love 8vim! 

Ps2 Is there some instruction about how to use gestures?

**Comment by flide** [comment, marked as answer]
With the new layers PR, we will soon have diacritics on the base layout itself, And I will try to improve the wiki.
